.travis.yml

```diff
@@ -57,7 +58,7 @@ matrix:
       # Disable compiler overrides as they cause problems
       - unset CC ; unset CXX ; unset CC_FOR_BUILD ; unset CXX_FOR_BUILD
       # homebrew addon is just broken
-      - HOMEBREW_NO_AUTO_UPDATE=1 brew install yasm
+      - HOMEBREW_NO_AUTO_UPDATE=1 brew install yasm json-c
```

Comment:
Why is this needed?

Reply:
from some failed try before ;) , I will fix it later